### PR TITLE
*: adjust metrics

### PR DESCRIPTION
**Pull request by overvenus**
* remove tikv_raftstore_proposal_total type "all"
 * rename some metrics in server/metrics.rs

PTAL @siddontang @BusyJay

### CODE REVIEW

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -528,7 +528,6 @@ impl Peer {
         }
 
         debug!("{} propose command with uuid {:?}", self.tag, cmd.uuid);
-        PEER_PROPOSAL_COUNTER_VEC.with_label_values(&["all"]).inc();
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: why remove `all`?

Comment (overvenus) [inline src/raftstore/store/peer.rs]: We can compute the total number of proposals by `sum(tikv_raftstore_proposal_total)`, `all` is unnecessary.

Comment (siddontang) [inline src/raftstore/store/peer.rs]: But `sum(tikv_raftstore_proposal_total)` contains no error.

**Comment by siddontang** [comment]
LGTM

File: src/server/metrics.rs [hunk 2]
```diff
@@ -27,14 +27,14 @@ lazy_static! {
             &["type"]
         ).unwrap();
 
-    pub static ref RECV_MSG_COUNTER: CounterVec =
+    pub static ref RECV_MSG_COUNTER_VEC: CounterVec =
```
Comment (BusyJay) [inline src/server/metrics.rs]: I personally don't like the naming style that attaches type name to the identifier.

**Pull request closed without merging**